main.cc

```diff
@@ -402,6 +402,10 @@ int main( int argc, char ** argv )
   app.setStyle(new GdAppStyle);
 #endif
 
+#if defined( Q_OS_LINUX ) && QT_VERSION >= QT_VERSION_CHECK( 5, 7, 0 )
+  app.setDesktopFileName( QStringLiteral( "org.goldendict.GoldenDict.desktop" ) );
```

Comment:
Please remove the `.desktop` extension. See https://codereview.qt-project.org/c/qt/qtbase/+/477797